### PR TITLE
fix name/url/email etc fields in operator bundle

### PR DESCRIPTION
**Pull request by cniackz**
### Objective:

To have just one CSV after renaming the package name in previous PR: [
fix package name in operator bundle generation](https://github.com/minio/directpv/pull/832/files)

### To fix:

```sh
$ operator-sdk bundle validate ./bundle
FATA[0000] [error loading objs in directory: invalid bundle: contains multiple CSVs, no bundle csv found, unable to load bundle from directory bundle/manifests] 
```

### CODE REVIEW

File: helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml [hunk 1]
```diff
@@ -318,14 +318,15 @@ spec:
...
   links:
-  - name: Directpv Operator
-    url: https://directpv-operator.domain
+  - name: Minio Directpv Operator Rhmp
+    url: https://minio-directpv-operator-rhmp.domain
   maintainers:
-  - email: cesar@minio.io
-    name: cesar
+  - email: your@email.com
+    name: Maintainer Name
```
Comment (harshavardhana) [inline helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:327]:
```suggestion
  - email: dev@minio.io
    name: MinIO, Inc.
```

File: helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml [hunk 2]
```diff
@@ -318,14 +318,15 @@ spec:
...
   maturity: alpha
   provider:
-    name: directpv
+    name: Provider Name
```
Comment (harshavardhana) [inline helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:330]:
```suggestion
    name: MinIO, Inc.
```

File: helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml [hunk 3]
```diff
@@ -318,14 +318,15 @@ spec:
...
+    url: https://your.domain
```
Comment (harshavardhana) [inline helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:331]:
```suggestion
    url: https://min.io
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by cniackz** [comment]
Done @harshavardhana changes committed, now waiting for tests to pass!. Thanks for the review 👍

File: helm/bundle.Dockerfile [hunk 4]
```diff
@@ -4,7 +4,7 @@ FROM scratch
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
-LABEL operators.operatorframework.io.bundle.package.v1=directpv-operator
+LABEL operators.operatorframework.io.bundle.package.v1=minio-directpv-operator-rhmp
```
Comment (balamurugana) [inline helm/bundle.Dockerfile:7]: Could you update `helm/generate.sh` script as well?

File: helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml [hunk 5]
```diff
@@ -75,8 +75,8 @@ spec:
     - kind: DirectpvChart
       name: directpvcharts.charts.quay.io
       version: v1alpha1
-  description: directpv
-  displayName: directpv
+  description: Minio Directpv Operator Rhmp description. TODO.
```
Comment (balamurugana) [inline helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:78]: May be `description: Minio DirectPV Operator manages DirectPV deployments.`

File: helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml [hunk 6]
```diff
@@ -75,8 +75,8 @@ spec:
...
+  displayName: Minio Directpv Operator Rhmp
```
Comment (balamurugana) [inline helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:79]: May be `displayName: Minio DirectPV Operator`

File: helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml [hunk 7]
```diff
@@ -318,14 +318,15 @@ spec:
...
+    url: https://minio-directpv-operator-rhmp.domain
   maintainers:
-  - email: cesar@minio.io
-    name: cesar
+  - email: dev@minio.io
+    name: MinIO, Inc.
   maturity: alpha
   provider:
-    name: directpv
+    name: MinIO, Inc.
```
Comment (balamurugana) [inline helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:330]: Do we need to have `  - name: MinIO, Inc.`?

File: helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml [hunk 8]
```diff
@@ -318,14 +318,15 @@ spec:
...
+    url: https://min.io
```
Comment (balamurugana) [inline helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:331]: May be `url: https://min.io/directpv`

File: helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml [hunk 9]
```diff
@@ -318,14 +318,15 @@ spec:
   - supported: true
     type: AllNamespaces
   keywords:
-  - directpv
+  - minio-directpv-operator-rhmp
   links:
-  - name: Directpv Operator
-    url: https://directpv-operator.domain
+  - name: Minio Directpv Operator Rhmp
+    url: https://minio-directpv-operator-rhmp.domain
```
Comment (balamurugana) [inline helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:324]: May be `url: https://min.io/directpv`

File: helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml [hunk 10]
```diff
@@ -318,14 +318,15 @@ spec:
   - supported: true
     type: AllNamespaces
   keywords:
-  - directpv
+  - minio-directpv-operator-rhmp
   links:
-  - name: Directpv Operator
-    url: https://directpv-operator.domain
+  - name: Minio Directpv Operator Rhmp
```
Comment (balamurugana) [inline helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:323]: We would need to style DirectPV as `DirectPV` like ` - name: Minio DirectPV Operator RHMP`

**Review by balamurugana — CHANGES_REQUESTED** [review]

File: helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml (hunk 7 above)
Comment (cniackz) [reply helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:330]: I am not sure, but by looking at other operator in the catalog, they do have it. Why? Is there a problem with having it?

Comment (cniackz) [reply helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:330]: Oh you mean MinIO versus DirectPV, well as long as we have something that should be good enough!. What should I put?

Comment (cniackz) [reply helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:330]: Ok in the meantime I will just put DirectPV and later we can modify if needed.

File: helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml [hunk 11]
```diff
@@ -63,10 +63,10 @@ metadata:
         }
       ]
     capabilities: Basic Install
-    createdAt: "2023-08-17T13:36:51Z"
-    operators.operatorframework.io/builder: operator-sdk-v1.31.0
+    createdAt: "2023-09-06T18:13:45Z"
+    operators.operatorframework.io/builder: operator-sdk-v1.28.0-ocp
```
Comment (Praveenrajmani) [inline helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:67]: Q: Did we downgrade the builder version?

**Review by Praveenrajmani — APPROVED** [review]

File: helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml [hunk 12]
```diff
@@ -318,14 +318,15 @@ spec:
...
   maintainers:
-  - email: cesar@minio.io
-    name: cesar
+  - email: dev@minio.io
+    name: MinIO, Inc.
   maturity: alpha
   provider:
-    name: directpv
+    name: DirectPV
+    url: https://min.io/directpv
```
Comment (balamurugana) [inline helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml:331]: Can you update `generate.sh` script for all these values?

File: helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml (hunk 11 above)
Comment (cniackz) [reply helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml]: Sorry no, that wasn't intended.

**Review by Praveenrajmani — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**